spec/models/like_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before(:each) do
+    @user = User.new(id: 1, name: 'Ben', bio: 'I am a content creator', photo: '', posts_counter: 0)
+    @post = Post.new(title: 'Post1', text: 'Text...', comments_counter: 0, likes_counter: 1, author_id: @user.id)
+    @like = Like.new(author_id: @user.id, post_id: @user.id)
+  end
+
+  describe 'validation tests' do
+    it 'validates the author_id is an integer' do
+      @like.author_id = 1
+      expect(@like.author_id).to eq(1)
+    end
+
+    it 'validates the post_id is an integer' do
+      @like.post_id = 1
+      expect(@like.post_id).to eq(1)
+    end
+  end
+end
```

Comment:
Good job working on the project but your test cases are not running this what im getting when i run your tests
![tests](https://user-images.githubusercontent.com/79506105/215287848-ac2c5870-f4a8-4d27-b19c-91428cb5a746.PNG) it would be nice if you corrected this and please kindly run  the tests on your end before you submit for another review.